File: PN5.cpp


#include<iostream>
using namespace std;

class Number
{
    private:
        int iNo;        

    public:
        
        void Accept()      
        {
            cout<<"Enter the value : "<<endl;
            cin>>this->iNo;
        }
		
		bool Display()
		{
		    if((iNo % 2) == 0)
			{
				return true;
			}
			else
			{
				return false;
			}
		}

       
};

int main()
{
    Number nobj;
    bool bRet = false;
	
    nobj.Accept();
    bRet = nobj.Display();

    if(bRet = true)
	{
		cout<<"The number is even"<<endl; 
	}
	else
	{
		cout<<"The number is odd"<<endl;
	}

    return 0;
}

File: PN5.java
//DisplayForward

import java.lang.*;
import java.util.*;

class Number
{
	private int iNo;
	
	public void Accept()
	{
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter number : ");
		this.iNo = sobj.nextInt();
	}
	
	public void DisplayForward()
	{
		int iCnt = 0;
		
		for(iCnt = 1; iCnt<=iNo;iCnt++)
		{
			System.out.println(iCnt);
		}
	
	}
	
}

class PN5
{
	public static void main(String arg[])
	{
		Number nobj = new Number();
		
		nobj.Accept();
		
		nobj.DisplayForward();
	}
}